core/core/src/main/java/androidx/core/graphics/drawable/IconCompat.java

```
@@ -598,15 +598,15 @@ private Drawable loadDrawableInner(Context context) {
                         BitmapFactory.decodeByteArray((byte[]) mObj1, mInt1, mInt2)
                 );
             case TYPE_URI:
-                InputStream is = getUriInputStream(context);
-                if (is != null) {
+            try (InputStream is = getUriInputStream(context)) {
                     return new BitmapDrawable(context.getResources(),
                             BitmapFactory.decodeStream(is));
+                } catch (Exception e) {
+                    /* ignore */
```

Nit: for consistency with the existing error logging in `loadDrawableInner()` (e.g. line 590), consider logging the exception instead of silently ignoring it:

```java
catch (Exception e) {
    Log.e(TAG, "Unable to load image from URI: " + getUri(), e);
}
```

This would make debugging easier while preserving current fallback behavior.

---

ok